### PR TITLE
Cybersecurity New Topic: Risk

### PR DESCRIPTION
**Pull request by jonchery**
### Description

Added new cybersecurity folder and concept, Risk

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/cybersecurity/concepts/risk/risk.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+Title: 'Risk'
+Description: 'Risk is the possibility or likelihood that a threat will exploit a vulnerability to cause harm to an asset.'
+Subjects:
+  - 'Computer Science'
+  - 'Cybersecurity'
+Tags:
+  - 'Cybersecurity'
```
Comment (SSwiniarski) [inline content/cybersecurity/concepts/risk/risk.md:8]: If we could, can we add at least one more tag?

File: content/cybersecurity/concepts/risk/risk.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  - 'paths/cyber-resilience-risk-management'
+  - 'fundamentals-of-cybersecurity'
+---
+
+**Risk** is the possibility or likelihood that a threat will exploit a vulnerability to cause harm to an asset. _Risks_ posed to people, processes or technology could result in financial loss, operational disruption or physical damage.
+
+## Risk In-Use:
+
+Risk can take several forms. External cyber risks include cyberattacks, phishing, ransomware, and DDoS attacks. Internal risks can come from malicious insider threats, however, they can also come as a result of employee negligence. Unpatched software, poor training, and misconfiguring databases are all examples of how employees can mistakenly expose information.
+
```
Comment (SSwiniarski) [inline content/cybersecurity/concepts/risk/risk.md:19]: Remove extra line break

File: content/cybersecurity/concepts/risk/risk.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+## Different Frameworks for Assessing Risk
+
+A cybersecurity framework provides a common set of standards for organizations to assess and understand their overall security postures. Here are a few of the most common cybersecurity frameworks:
+
+- NIST (National Institute of Standards and Technology): This was established by President Obama in an effort to better define and understand cyber risk. NIST is probably the most recognizable and widely used framework for assessing cybersecurity maturity, identifying security gaps, and meetings cybersecurity regulations.
```
Comment (SSwiniarski) [inline content/cybersecurity/concepts/risk/risk.md:25]: Remove the line breaks between the bullet points.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@jonchery I did a first review and only had a couple of comments.

**Review by jonchery — COMMENTED** [review]
Completed request

**Review by SSwiniarski — APPROVED** [review]
@jonchery Looks good for a second review.

**Comment by SSwiniarski** [comment]
@jonchery, one more edit. Can we remove the trailing punctuation from the headers? The colon `:` will cause this to fail one of the checks.

**Review by jonchery — COMMENTED** [review]
Removed ":"

File: content/cybersecurity/concepts/risk/risk.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+**Risk** is the possibility or likelihood that a threat will exploit a vulnerability to cause harm to an asset. _Risks_ posed to people, processes or technology could result in financial loss, operational disruption or physical damage.
+
+## Risk In-Use
+
+Risk can take several forms. External cyber risks include cyberattacks, phishing, ransomware, and DDoS attacks. Internal risks can come from malicious insider threats, however, they can also come as a result of employee negligence. Unpatched software, poor training, and misconfiguring databases are all examples of how employees can mistakenly expose information.
+
+## Different Frameworks for Assessing Risk
+
+A cybersecurity framework provides a common set of standards for organizations to assess and understand their overall security postures. Here are a few of the most common cybersecurity frameworks:
```
Comment (yangc95) [inline content/cybersecurity/concepts/risk/risk.md:23]:
```suggestion
A cybersecurity framework provides a common set of standards for organizations to assess and understand their overall security postures. Here are a few of the most common cybersecurity frameworks:

```

File: content/cybersecurity/concepts/risk/risk.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+- NIST (National Institute of Standards and Technology): This was established by President Obama in an effort to better define and understand cyber risk. NIST is probably the most recognizable and widely used framework for assessing cybersecurity maturity, identifying security gaps, and meetings cybersecurity regulations.
```
Comment (yangc95) [inline content/cybersecurity/concepts/risk/risk.md:24]: Do these have official sites?

File: content/cybersecurity/concepts/risk/risk.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+  - 'Cybersecurity'
+  - 'Business'
+CatalogContent:
+  - 'paths/cyber-resilience-risk-management'
+  - 'fundamentals-of-cybersecurity'
+---
+
+**Risk** is the possibility or likelihood that a threat will exploit a vulnerability to cause harm to an asset. _Risks_ posed to people, processes or technology could result in financial loss, operational disruption or physical damage.
+
+## Risk In-Use
```
Comment (yangc95) [inline content/cybersecurity/concepts/risk/risk.md:17]: Can we edit the heading to better reflect the info below?

**Review by yangc95 — COMMENTED** [review]
@jonchery I left a few comments

**Comment by SSwiniarski** [comment]
@jonchery, in the interests of getting this merged, I applied the above edits. Once @yangc95 reviews and approves, we can merge this.

**Comment by yangc95** [comment]
@jonchery live link: https://www.codecademy.com/resources/docs/cybersecurity/risk

Screenshot 2023-03-20 at 6 10 41 PM

cc: @SSwiniarski

**Pull request merged**